.gitignore

```diff
@@ -6,9 +6,11 @@ __pycache__/
 back-end/pkg
 back-end/bin
 back-end/src/*
+back-end/resources/room_config.json
```

Comment:
why not ignore all other jsons as well?

Reply:
Now we ignore all JSON files in /production

Reply:
renaming room_config to default would result in default_config.json.default
this does not seem more intuitive than room_config.json(.default), maybe just config.json(.default)